.ci/integration_test.groovy

```diff
@@ -185,18 +218,28 @@ def call(ghprbActualCommit, ghprbCommentBody, ghprbPullId, ghprbPullTitle, ghprb
                     sh 'curl -s https://codecov.io/bash | bash -s - -t @CODECOV_TOKEN'
                 }
             }
-    
-            def run_intergration_test = { chunk_suffix, run_test ->
+
+            groovy.lang.Closure run_intergration_test = { chunk_suffix, run_test ->
                 node("test_java") {
                     println "${NODE_NAME}"
                     container("java") {
                         deleteDir()
                         unstash 'binaries'
                         unstash 'tispark'
+                        if (TEST_TIFLASH != "false") {
+                            unstash 'tiflash_binary'
+                        }
     
                         try {
 
-                            if(TIDB_BRANCH.contains("4.0") && TIKV_BRANCH.contains("4.0") && PD_BRANCH.contains("4.0")) {
+                            groovy.lang.Closure isv4 = { branch_name ->
+                                if (branch_name.startsWith("v4") || branch_name == "master") {
```

Comment:
```suggestion
                                if (branch_name.startsWith("v4") || branch_name.startsWith("release-4") || branch_name == "master") {
```

Reply:
can we use tag to specify tidb branch name?